index.css

```diff
@@ -1,361 +1,179 @@
-    width: 406px;
-    height: 522px;
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
-    margin-top: 20px;
-}
-
-.grid-item6{
-    width: 406px;
-    height: 522px;
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
-    margin-top: 20px;
-}
-.grid-item7{
-    width: 406px;
-    height: 522px;
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
-    margin-top: 20px;
-}
-.grid-item8{
-    width: 406px;
-    height: 522px;
-    border: 1px solid orange;
-    border-radius: 30px 30px 0px 0px;
-    margin-top: 20px;
-}
-.grid-item9{
-    width: 406px;
-    height: 200px;
-    margin-top: 20px;
-    
-}
-
-
-.grid-item10{
-    width: 406px;
-    height: 200px;
-    margin-top: 20px;
-}
-.footer1{
-    width:1182px;
+  }
+  
+  /* Footer */
+  .footer1 {
+    width: 1182px;
     background-color: white;
     margin: auto;
     height: 270px;
-    
-}
-.footer2{
+  }
+  
+  .footer2 {
     width: 1137px;
     height: 240px;
-    border: 1px solid #25ac45;
+    border: 1px solid var(--primary-color);
     margin: auto;
     display: flex;
-    
-}
-.footer3 img{
-    width:210px ;
-    height: 66px ;
-    padding-bottom: 10px;
-
-}
-.footer3{
-    width: 100%;
-    padding: 10px 10px 10px;
-    overflow: hidden;
-}
-.footer30{
-    float: left;
-}
-.footer3 a{
-text-decoration: none;
-color: orange;
-}
-.footer4 {
-    font-size: 14px;
-    margin-left: 260px;
-
-}
-.footer5{
+  }
+  
+  .footer3 img {
+    width: 210px;
+    height: 66px;
     padding-bottom: 10px;
-}
-.end-game{
+  }
+  
+  .footer3 a {
+    text-decoration: none;
+    color: var(--border-color);
+  }
+  
+  .end-game {
     width: 1182px;
     height: 170px;
-    margin: auto;
+    margin: 30px auto;
     border: 1px solid #b4bdb6;
-    margin-top: 30px;
     border-radius: 15px;
     display: flex;
     padding-top: 10px;
-    margin-bottom: 30px;
-}
-.end-game1{
-    padding-left: 20px;
-    padding-right: 100px;
-}
-.end-game li a{
+  }
+  
+  .end-game li a {
     text-decoration: none;
-    color: black;
+    color: var(--text-color);
     font-size: 15px;
-}
-.end-game li{
-padding-bottom: 5px;
-}
-.the-end{
-    text-align: center;
-    background-color: #13883C;
-    width: 100%;
-    line-height: 40px;
-    color: white;
-}
-.grid-item22{
-    
-}
-.grid-item22 img:hover{
-    transition: 2s all;
-   width: 110px;
-   height: 85px;
-}
-.header{
+  }
+  
+  /* Header */
+  .header {
     width: 1349px;
     height: 120px;
     background-color: #fadc68;
-    background-image: url(/img2.jpg/migrinheader\ \(3\).png);
+    background-image: url('/img2.jpg/migrinheader.png');
     background-position: right bottom, left top;
     background-repeat: no-repeat, repeat;
-    
-}
-.header{
     display: flex;
-    vertical-align: middle;
-}
-
-.header1{
-padding: 13px;
- margin-left: 91px;
- margin-right: 350px;
-display: flex;
-}
-.header2{
-padding-top: 20px;
-display: flex;
-}
+    align-items: center;
+  }

```

Comment:
Inconsistent background-image URL format. Consider standardizing the format for consistency and avoiding potential path issues.

```diff
-  background-image: url('/img2.jpg/migrinheader.png');
+  background-image: url('/img2.jpg/migrinheader\ \(3\).png');
```